zaza/openstack/charm_tests/test_utils.py

```diff
@@ -89,7 +89,53 @@ def audit_assertions(action,
             assert value == "PASS", "Unexpected failure: {}".format(key)
 
 
-class BaseCharmTest(unittest.TestCase):
+class CharmTestAssertions:
+    """Custom assertions to support zaza testings."""
+
+    def assertActionRanOK(self, action):
+        """Assert that the remote action ran successfully.
+
+        Example usage::
+
+            self.assertActionRanOK(model.run_action(
+                unit,
+                'pause',
+                model_name=self.model_name))
+
+            self.assertActionRanOK(model.run_action_on_leader(
+                unit,
+                'pause',
+                model_name=self.model_name))
+
+        :param action: Action object to check.
+        :type action: juju.action.Action
+        :raises: AssertionError if the assertion fails.
+        """
+        print("Checking action")
+        if action.status != 'completed':
+            msg = ("Action '{name}' exited with status '{status}': "
+                   "'{message}'").format(**action.data)
+            raise AssertionError(msg)
+
+    def assertRemoteRunOK(self, run_output):
```

Comment:
The function doesn't use self.  It can be a `@staticmethod`.

Reply:
I'll move them